### PR TITLE
Stats: add Tracks events to actions available after publishing post/page

### PR DESCRIPTION
**Pull request by aaronyan**
Adding stat tracking for actions available after a user publishes/updates a post/page. This is helpful for us to understand what user's are doing after they finishing publishing.

### CODE REVIEW

File: client/components/web-preview/toolbar.jsx [hunk 1]
```diff
@@ -93,7 +104,7 @@ class PreviewToolbar extends Component {
 						aria-label={ translate( 'Close preview' ) }
 						className="web-preview__close"
 						data-tip-target="web-preview__close"
-						onClick={ onClose }
+						onClick={ onClose } // **HERE**
```
Comment (aaronyan) [inline client/components/web-preview/toolbar.jsx]: Anyone know how to add a Tracks event here? I tried adding my own function that calls both `onClose ` and `analytics.tracks.recordEvent`, but no dice. It records the Tracks event but does not close the preview.

**Comment by aidvu** [comment]
We try to use actions for tracking now. e.g. https://github.com/Automattic/wp-calypso/pull/16816

File: client/components/web-preview/toolbar.jsx [hunk 2]
```diff
@@ -82,6 +87,12 @@ class PreviewToolbar extends Component {
 			translate,
 		} = this.props;
 
+		// @todo: having trouble adding a Tracks event to the onClose event (see **HERE**)
```
Comment (aidvu) [inline client/components/web-preview/toolbar.jsx]: Try to make a separate function outside:
```
handleClosePreview = () => {
    analytics.tracks.recordEvent( 'calypso_editor_preview_close_click' );
    this.props.onClose();
};
```

Comment (aaronyan) [reply client/components/web-preview/toolbar.jsx]: Wooohoo! Thank you for your help @aidvu. Ah, makes sense I should have called `onClose()` directly from `this.props.`

Comment (aidvu) [reply client/components/web-preview/toolbar.jsx]: Anytime. :)

File: client/components/clipboard-button-input/index.jsx [hunk 3]
```diff
@@ -84,4 +87,10 @@ class ClipboardButtonInputExport extends React.Component {
 	}
 }
 
-export default localize( ClipboardButtonInputExport );
+const wpcomEditorToolbarWebPreviewClipboardUrlButtonClick = () => {
```
Comment (aidvu) [inline client/components/clipboard-button-input/index.jsx]: No need for external definitions and `composeAnalytics`. We use that only if you have multiple events. :)

File: client/components/clipboard-button-input/index.jsx [hunk 4]
```diff
@@ -84,4 +87,10 @@ class ClipboardButtonInputExport extends React.Component {
...
+	composeAnalytics( recordTracksEvent( 'calypso_editor_clipboard_url_button_click', {} ) );
+};
+
+export default connect( null, {
+	wpcomEditorToolbarWebPreviewClipboardUrlButtonClick,
```
Comment (aidvu) [inline client/components/clipboard-button-input/index.jsx]: Just pass `recordTracksEvent` here.

File: client/components/clipboard-button-input/index.jsx [hunk 5]
```diff
@@ -54,6 +56,7 @@ class ClipboardButtonInputExport extends React.Component {
 				isCopied: false,
 			} );
 		}, 4000 );
+		this.props.wpcomEditorToolbarWebPreviewClipboardUrlButtonClick();
```
Comment (aidvu) [inline client/components/clipboard-button-input/index.jsx]: You can do 
```
this.props.recordTracksEvent( 'calypso_editor_clipboard_url_button_click', {} );```

Comment (aaronyan) [reply client/components/clipboard-button-input/index.jsx]: lol, oppsies. simplifying to this.

**Comment by aidvu** [comment]
Tests left, and LGTM. :)

**Comment by aidvu** [comment]
Thanks @bluefuton! Didn't have the time to check why the tests were failing!

**Comment by bluefuton** [comment]
No worries @aidvu! Seemed like a prime time to use Jest snapshots :)

**Comment by aaronyan** [comment]
Thank you to both of you! Appreciate all the help @aidvu with using actions for Tracks events and @bluefuton for fixing up my broken tests! 🥇 🥇 🥇

**Pull request merged**